packages/popup/src/pages/AdvancedGas/index.js

```diff
@@ -0,0 +1,305 @@
+      ) || estimateGasLimit,
+    nonce: formatDecimalToHex(inputNonce || advancedGasSetting.nonce || nonce),
+    storageLimit: formatDecimalToHex(
+      advancedGasSetting.storageLimit || storageLimit,
+    ),
+  }
+  if (!params.maxFeePerGas) delete params.maxFeePerGas
+  if (!params.maxPriorityFeePerGas) delete params.maxPriorityFeePerGas
+  if (!params.gasPrice) delete params.gasPrice
+  if (!params.storageLimit) delete params.storageLimit
+  if (!params.nonce) delete params.nonce
+
+  const estimateRst = useEstimateTx(params) || {}
+  const {gasUsed} = estimateRst
+  const {
+    gasPrice: advancedGasPrice,
+    maxFeePerGas: advancedMaxFeePerGas,
+    maxPriorityFeePerGas: advancedMaxPriorityFeePerGas,
+  } = advancedGasSetting
+
+  useEffect(() => {
+    !isTxTreatedAsEIP1559 &&
+      !inputGasPrice &&
+      setInputGasPrice(
+        advancedGasPrice
+          ? convertDecimal(advancedGasPrice, 'divide', GWEI_DECIMALS)
+          : convertDataToValue(suggestedGasPrice, GWEI_DECIMALS),
+      )
+    // gas station unit is GWei
+    isTxTreatedAsEIP1559 &&
+      !inputMaxFeePerGas &&
+      setInputMaxFeePerGas(
+        selectedGasLevel === 'advanced'
+          ? convertDecimal(advancedMaxFeePerGas, 'divide', GWEI_DECIMALS)
+          : suggestedMaxFeePerGas,
+      )
+    isTxTreatedAsEIP1559 &&
+      !inputMaxPriorityFeePerGas &&
+      setInputMaxPriorityFeePerGas(
+        selectedGasLevel === 'advanced'
+          ? convertDecimal(
+              advancedMaxPriorityFeePerGas,
+              'divide',
+              GWEI_DECIMALS,
+            )
+          : suggestedMaxPriorityFeePerGas,
+      )
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [
+    selectedGasLevel,
+    isTxTreatedAsEIP1559,
+    suggestedGasPrice,
+    suggestedMaxFeePerGas,
+    suggestedMaxPriorityFeePerGas,
+    advancedGasPrice,
+    advancedMaxPriorityFeePerGas,
+    advancedMaxFeePerGas,
+  ])
+
+  const onChangeGasPrice = gasPrice => {
+    setInputGasPrice(gasPrice)
+    if (!networkTypeIsCfx) {
+      if (new Big(gasPrice || '0').times('1e9').gt(0)) {
+        setGasPriceErr('')
+      } else {
+        setGasPriceErr(
+          t('gasPriceErr', {
+            amount: 0.000000001,
+            unit: 'GWei',
+          }),
+        )
+      }
+    } else {
+      if (new Big(gasPrice || '0').gt(0)) {
+        setGasPriceErr('')
+      } else {
+        setGasPriceErr(
+          t('gasPriceErr', {
+            amount: 1,
+            unit: 'GDrip',
+          }),
+        )
+      }
+    }
+  }
+
+  const onChangeMaxFeePerGas = maxFeePerGas => {
+    console.log('maxFeePerGas', maxFeePerGas)
+    setInputMaxFeePerGas(maxFeePerGas)
+    if (
+      !maxPriorityFeePerGasErr &&
+      new Big(inputMaxPriorityFeePerGas).gt(maxFeePerGas || '0')
+    ) {
+      setMaxPriorityFeePerGasErr(t('maxPriorityFeePerGasHighErr'))
+    } else if (
+      maxPriorityFeePerGasErr === t('maxPriorityFeePerGasHighErr') &&
```

Comment:
I think maybe ` maxPriorityFeePerGasErr === t('maxPriorityFeePerGasHighErr')` is not required

Reply:
1. error will always display under max priority fee input
2. display max priority fee input error first, then check maxFee need >= max priority fee
According above, it's reqiured